Makefile

```diff
@@ -20,11 +20,10 @@ main-image: .main-image
 	touch .main-image
 
 setup-image: .setup-image
-.setup-image: scripts/setup.Containerfile scripts/setup.sh
+.setup-image: scripts/setup.Containerfile scripts/setup.sh $(shell find ./scripts -name "*.ts" -not -path "*/node_modules/*")
```

Comment:
I'm not really sure what this does? Rebuild the image if the node_modules changed?

Reply:
The find command lists all .ts files in the scripts folder which are not within a node_modules folder. I.e. it rebuilds the setup image if any of the scripts that it contains changes.